libbeat/cmd/instance/beat.go

```diff
@@ -554,6 +555,8 @@ func (b *Beat) Setup(settings Settings, bt beat.Creator, setup SetupSettings) er
 		}
 
 		if setup.MachineLearning && b.SetupMLCallback != nil {
+			cfgwarn.Deprecate("8.0", "Setting up ML using Filebeat is going to be removed. Please use the ML app to setup jobs.")
```

Comment:
```suggestion
			cfgwarn.Deprecate("8.0", "Setting up ML using %v is going to be removed. Please use the ML app to setup jobs.", strings.Title(b.Info.Beat))
```

Reply:
Also, might be more explicit to use `8.0.0` instead of `8.0`? I realize we are inconsistent about this in our codebase so if you want to keep it `8.0` over here, that's fine.